Assets/Scripts/Contracts/PepemonMatchmaker.cs

```diff
@@ -41,7 +48,23 @@ public static async Task<string> GetDeckOwner(PepemonLeagues league, ulong deckI
         BlockParameter from,
         BlockParameter to)
     {
-        var eventLogs = await new BattleFinishedEventDTO()
+        /*if (Utils.IsWebGLBuild())
+        {
+            Thirdweb.Contract contract = contracts(league);
+            var events = await contract.Events.GetAll(
+                new EventQueryOptions
+                {
+                    fromBlock = (int)from.BlockNumber.Value,
+                    toBlock = (int)to.BlockNumber.Value,
+                    filters = new Dictionary<string, object>
+                    {
+                        ["winner"] = asWinner ? playerAddress : null,  // address winner
+                        ["loser"] = asWinner ? null : playerAddress,   // address loser
+                    }
+                });
+            //events.Last().data todo: cast to event type
+        }*/
+        var eventLogs = await new BattleFinishedEventDto()
             .GetEventABI()
             .CreateFilterInput(
                 Addresses[(int)league],
```

Comment:
Is the WebGL code not needed here?

Reply:
this method is not being used right now anywhere actuallym we only use `WaitForCreatedBattle` in PepemonBattle.cs for waiting for battles to be created.

I dont remember why this method was created, im gonna remove it